js/test/strings-ext/testlistfmt_en.js

```diff
@@ -50,7 +50,7 @@ module.exports.testlistfmt_en = {
     
         test.expect(2);
         test.ok(fmt !== null);
-        test.equal(fmt.format(["one", "two"]), "one and two");
```

Comment:
It looks like CLDR changed the format for the "short" length to be a tight, small list. I don't think that is appropriate for the default length for this class. I think we should change the default length to be "medium" or "long" instead. If we do that, then many of these unit tests do not need to change.

Reply:
I'm going to change as `medium`.  nonetheless, both `medium' and `long` cause 200+ fail cases.